pyvista/plotting/plotting.py

```diff
@@ -1745,6 +1749,11 @@ def prepare_mapper(scalars):
             elif isinstance(clim, float) or isinstance(clim, int):
                 clim = [-clim, clim]
 
+            if log_scale:
+                if clim[0] <= 0:
+                    clim = [sys.float_info.min, clim[1]]
```

Comment:
For my edification, why does this have to greater than 0 vs just 0?

Reply:
When `clim[0]` is 0, `log(0)` is `-inf`, and VTK produces:

![image](https://user-images.githubusercontent.com/1608652/108102197-ff789600-7066-11eb-8432-b2282e1c5f40.png)